quill-async/src/main/scala/io/getquill/context/async/AsyncContextConfig.scala

```diff
@@ -19,14 +20,19 @@ abstract class AsyncContextConfig[C <: Connection](
   def database = Try(config.getString("database")).toOption
   def port = config.getInt("port")
   def host = config.getString("host")
+  def sslProps = Map(
+    "sslmode " -> Try(config.getString("sslmode")).toOption,
+    "sslrootcert" -> Try(config.getString("sslrootcert")).toOption
+  ).collect { case (key, Some(value)) => key -> value }
 
   def configuration =
```

Comment:
Could you create a new unit test `AsyncContextConfigSpec.scala` and test this method?